winml/lib/Api.Core/inc/WinMLAdapter.h

```diff
@@ -52,4 +57,18 @@ class OrtObject {
 using ModelProto = OrtObject<onnx::ModelProto>;
 using IOBinding = OrtObject<onnxruntime::IOBinding>;
 
+class LotusEnvironment {
+ public:
+  LotusEnvironment() {
+    lotusEnvironment = WinML::Adapter::CreateLotusEnvironment();
```

Comment:
>WinML::Adapter [](start = 23, length = 14)

_winmla

Reply:
which you don't really need here since you area already inside that namespace

---
In reply to: [345522968](https://github.com/microsoft/onnxruntime/pull/2381#discussion_r345522968) [](ancestors = 345522968)